### PR TITLE
dockerize and minor changes

### PR DESCRIPTION
**Pull request by brgvgns**

### CODE REVIEW

**Comment by amahadaya** [comment]
+1 on code, but want to wait for approval until docsite changes are complete.

**Comment by amahadaya** [comment]
+1 on code, will approve shortly

**Review by ngarg-vail — APPROVED** [review]

**Review by amahadaya — APPROVED** [review]

**Pull request merged**
